app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -305,7 +305,7 @@ private void showFatalPlayServiceMessage(@NonNull ConnectionResult result) {
     }
 
     private boolean isFatalPlayServiceError(int errorCode) {
-        return !GoogleApiAvailability.getInstance().isUserResolvableError(errorCode);
```

Comment:
But this works for me
On 21 Mar 2016 6:53 pm, "Said Tahsin Dane" <notifications@github.com> wrote:

> In app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java
> <https://github.com/gdg-x/frisbee/pull/608#discussion_r56866018>:
>
> > @@ -305,7 +305,7 @@ private void showFatalPlayServiceMessage(@NonNull ConnectionResult result) {
> >      }
> >
> >      private boolean isFatalPlayServiceError(int errorCode) {
> > -        return !GoogleApiAvailability.getInstance().isUserResolvableError(errorCode);
>
> I don't understand this change. The previous version was more meaningful
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/608/files/e5e01a74fed431d52c8a68c310c541bab2d31151#r56866018>
>